docs/tutorials/scheduling.rst

```diff
@@ -0,0 +1,269 @@
+.. code-block:: python
+
+    >>> from astroplan.plots import plot_schedule_airmass
+    >>> import matplotlib.pyplot as plt
+
+    >>> plt.figure(figsize = (14,6))
+    >>> plot_schedule_airmass(priority_schedule)
+    >>> plt.legend(loc = 1)
+    >>> plt.show()
+
+.. plot::
+
+    # first import everything we will need for the scheduling
+    import astropy.units as u
+    from astropy.time import Time
+    from astroplan import (Observer, FixedTarget, ObservingBlock, Transitioner, PriorityScheduler,
+                           Schedule)
+    from astroplan.constraints import AtNightConstraint, AirmassConstraint, MoonIlluminationConstraint
+    from astroplan.plots import plot_schedule_airmass
+    import matplotlib.pyplot as plt
+
+    # Now we define the targets, start time, end time, and observer.
+    Deneb = FixedTarget.from_name('deneb')
+    M13 = FixedTarget.from_name('m13')
+
+    start_time = Time('2016-06-07 02:00')
+    end_time = Time('2016-06-07 08:00')
+    apo = Observer.at_site('apo')
+
+    # Then define the constraints (global and specific) and make a list of the
+    # observing blocks that you want scheduled
+    global_constraints = [AirmassConstraint(max = 3, boolean_constraint = False),
+                          AtNightConstraint()]
+    rot = 20 * u.second
+    blocks = []
+    for filter in ['B', 'G', 'R']:
+        if filter == 'B':
+            constraints = [MoonIlluminationConstraint(max = 0.25)]
+        else:
+            constraints = None
+        blocks.append(ObservingBlock.from_exposures(Deneb, 1, 60*u.second, 17, rot,
+                                                    configuration = {'filter': filter},
+                                                    constraints = constraints))
+        blocks.append(ObservingBlock.from_exposures(M13, 0, 100*u.second, 17, rot,
+                                                    configuration = {'filter': filter},
+                                                    constraints = constraints))
+
+    # Define how the telescope transitions between the configurations defined in the
+    # observing blocks (target, filter, instrument, etc.).
+    transitioner = Transitioner(.8*u.deg/u.second,
+                                {'filter':{('B','G'): 10*u.second,
+                                           ('G','R'): 10*u.second,
+                                           'default': 30*u.second}})
+
+    # Initialize the scheduler
+    prior_scheduler = PriorityScheduler(constraints = global_constraints,
+                                        observer = apo, transitioner = transitioner)
+    # Create a schedule for the scheduler to insert the blocks into, and run the scheduler
+    schedule = Schedule(start_time, end_time)
+    priority_schedule = prior_scheduler(blocks, schedule)
+
+    # To get a plot of the airmass vs where the blocks were scheduled
+    plt.figure(figsize = (14,6))
+    plot_schedule_airmass(priority_schedule)
+    plt.legend(loc=1)
+    plt.show()
```

Comment:
The final plot has three legend labels for M13 and three labels for Deneb. Is it possible to make it only add labels once per target name?